batcher/aligned/src/main.rs

```diff
@@ -528,6 +596,51 @@ async fn main() -> Result<(), AlignedError> {
                 }
             }
         }
+        GetUserNonceFromEthereum(args) => {
+            let address = H160::from_str(&args.address).unwrap();
+            let network = args.network.into();
+            match get_nonce_from_ethereum(&args.eth_rpc_url, address, network).await {
+                Ok(nonce) => {
+                    info!(
+                        "Nonce for address {} in BatcherPaymentService contract is {}",
+                        address, nonce
+                    );
+                }
+                Err(e) => {
+                    error!("Error while getting nonce: {:?}", e);
+                    return Ok(());
+                }
+            }
+        }
+        GetAmountOfProofsInBatcherQueue(args) => {
+            let address = H160::from_str(&args.address).unwrap();
+            let network = args.network.into();
+            let ethereum_nonce =
+                match get_nonce_from_ethereum(&args.eth_rpc_url, address, network).await {
+                    Ok(nonce) => nonce,
+                    Err(e) => {
+                        error!("Error while getting nonce: {:?}", e);
+                        return Ok(());
+                    }
+                };
+            let batcher_nonce = match get_nonce_from_batcher(&args.batcher_url, address).await {
+                Ok(nonce) => nonce,
+                Err(e) => {
+                    error!("Error while getting nonce: {:?}", e);
+                    return Ok(());
+                }
+            };
+            if ethereum_nonce > batcher_nonce {
+                error!("User {} is in an invalid state.", address);
```

Comment:
Is this check necessary?

Reply:
Also, if this were to happen, what is there to do for the user?

Reply:
Maybe tell them to report a bug or something.

Reply:
I'm in favor of removing this as its unclear what actionable information it conveys for the user.